content/en/docs/started/installing-kubeflow.md

```diff
@@ -79,7 +79,7 @@ The following table lists <b>active distributions</b> that have <b>had a recent
           <a href="https://googlecloudplatform.github.io/kubeflow-gke-docs">Website</a>
         </td>
         <td>
-          1.6.1 <sup>[<a href="https://googlecloudplatform.github.io/kubeflow-gke-docs/docs/changelog/#161">Release Notes</a>]</sup>
+          1.7.0 <sup>[<a href="https://googlecloudplatform.github.io/kubeflow-gke-docs/docs/changelog/#171">Release Notes</a>]</sup>
```

Comment:
I think the change log URL tag should be `#170` instead of `#171`

Reply:
Thank you1 Fixed.